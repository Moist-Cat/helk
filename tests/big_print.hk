let y = 5;
let x = y + 3 in 
    print(prints("hello world") + x);
