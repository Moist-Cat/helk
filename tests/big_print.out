hello world
20.000000